src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/RuneOfUnemployment.java

```diff
@@ -0,0 +1,79 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.magical;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.bukkit.GameMode;
+import org.bukkit.Location;
+import org.bukkit.Particle;
+import org.bukkit.entity.Villager;
+import org.bukkit.entity.Villager.Profession;
+import org.bukkit.entity.WanderingTrader;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.Merchant;
+import io.github.thebusybiscuit.cscorelib2.inventory.ItemUtils;
+import io.github.thebusybiscuit.slimefun4.core.handlers.EntityInteractHandler;
+import io.github.thebusybiscuit.slimefun4.implementation.items.SimpleSlimefunItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.Objects.handlers.ItemHandler;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+
+/**
+ * This {@link SlimefunItem} allows you to reset a {@link Villager} profession. 
+ * Useful to reset a villager who does not have desirable trades.
+ *
+ * @author dNiym
+ *
+ * @see ItemHandler
+ *
+ */
+public class RuneOfUnemployment extends SimpleSlimefunItem<EntityInteractHandler> {
+
+    public RuneOfUnemployment(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
+        super(category, item, recipeType, recipe);
+    }
+ 
+    @Override
+    public EntityInteractHandler getItemHandler() {
+        return (p, entity, item, offhand) -> {
+            if (!(entity instanceof WanderingTrader) && entity instanceof Merchant) {
```

Comment:
Why not do (entity instance of Villager) instead of this weird double-check.
Should Mojang decide to add more types of Merchants, it would break too. So a Villager instanceof check is most appropriate here.

Especially if you are casting this to Villager anyway, since that would otherwise result in unexpected class casting exceptions.

Reply:
Hmmm I was pretty sure I did this for a reason since the merchant/villager api is so flaky.  Maybe it was that I checked to merchant because a regular villager will pass the villager instanceof check but not have any trades or a job.    A merchant object however will have a job but wandering traders can’t change profession because they’re not villagers.  Anyways I suppose I could just check to make sure they have a job before resetting.     I’ll do some testing to make sure it won’t cause issues!

Reply:
upon further testing it appears to be fine with wandering traders.